src/test/java/uk/gov/hmcts/cmc/claimstore/services/ResponseDeadlineCalculatorTest.java

```diff
@@ -22,6 +22,7 @@
 public class ResponseDeadlineCalculatorTest {
 
     private static final int DAYS_FOR_RESPONSE = 14;
+    private static final int DAYS_FOR_CLAIMANT_RESPONSE = 21;
```

Comment:
Can we please use same value in our tests as `claimantResponseDays` in application.yaml to avoid any confusion.